pkg/relayer/proxy/sync.go

```diff
@@ -375,6 +389,48 @@ func (server *relayMinerHTTPServer) serveSyncRequest(
 	return relayRequest, nil
 }
 
+// serviceConfigTypeDefault indicates that the service config being used is
+// the default service config, as opposed to an RPC-type specific config.
+const serviceConfigTypeDefault = "DEFAULT_SERVICE_CONFIG"
```

Comment:
We're ALWAYS returning the default in the function below.

Is this necessary? Not really sure of the intention anymore.

Reply:
The default is not always returned. If the request uses the RPC-type service-specific config then it returns:
` fmt.Sprintf("%s_SERVICE_CONFIG", rpcType.String())`

I've moved this to its own var with a comment to make it clearer.